tools/Makefile

```diff
@@ -328,6 +328,12 @@ endif
 clean::
 	rm -f -- caml-tex caml-tex.exe caml_tex.cm?
 
+# Debug printer script
+debug_printers: debug_printers.ml debug_printers.cmo
+	echo 'load_printer "tools/$(basename $<).cmo"' > '$@'
+	awk '{ print "install_printer Debug_printers." $$2 }' \
```

Comment:
I don't know `awk`, but my interpretation from this line is that it's slurping the lines from `debug_printers.ml` and looking at the second word of each line. This means that an editor of `debug_printers.ml` now needs to be careful to use exactly the same formatting as previous lines. In addition, if there is not an existing printer that suits someone's needs, they will not be able to add extra facilities here. Maybe that's all OK, but a loud comment in `debug_printers.ml` would seem to save a future person lots of head-scratching.

Reply:
Comment added; as discussed with you, there's already an `awk` dependency in the testsuite stuff, so we're not adding new requirements